### PR TITLE
Add int to datetime convert in update_subtask

### PR DESCRIPTION
**Pull request by pawelkisielewicz**
Resolves https://github.com/golemfactory/concent/issues/195

### CODE REVIEW

File: concent_api/core/tests/test_unit_message_handlers.py [hunk 1]
```diff
@@ -69,6 +77,30 @@ def test_that_messages_are_stored_with_correct_timestamps(self):
...
+        update_subtask(
+            subtask=subtask,
+            state=Subtask.SubtaskState.FORCING_RESULT_TRANSFER,
+            next_deadline=int(self.task_to_compute.compute_task_def['deadline']) + settings.CONCENT_MESSAGING_TIME,
+            force_get_task_result=self.force_get_task_result,
+            set_next_deadline=True,
+        )
+        subtask_state = Subtask.objects.get(subtask_id=self.task_to_compute.compute_task_def['subtask_id']).state
+        self.assertEqual(subtask_state, Subtask.SubtaskState.FORCING_RESULT_TRANSFER.name)
+
```
Comment (kbeker) [inline concent_api/core/tests/test_unit_message_handlers.py]: Please also make a test which will update subtask without next_deadline (passing to passive state)

Comment (pawelkisielewicz) [reply concent_api/core/tests/test_unit_message_handlers.py]: done

File: concent_api/core/tests/test_unit_message_handlers.py [hunk 2]
```diff
@@ -69,6 +77,53 @@ def test_that_messages_are_stored_with_correct_timestamps(self):
             parse_iso_date_to_timestamp(self.report_computed_task_timestamp)
         )
 
+    def test_that_update_subtask_passes_validations_from_passive_to_active_state(self):
```
Comment (dybi) [inline concent_api/core/tests/test_unit_message_handlers.py]: please add assertions about `next_deadline`

File: concent_api/core/tests/test_unit_message_handlers.py [hunk 3]
```diff
@@ -69,6 +77,53 @@ def test_that_messages_are_stored_with_correct_timestamps(self):
...
+            subtask=subtask,
+            state=Subtask.SubtaskState.FORCING_RESULT_TRANSFER,
+            next_deadline=int(self.task_to_compute.compute_task_def['deadline']) + settings.CONCENT_MESSAGING_TIME,
+            force_get_task_result=self.force_get_task_result,
+            set_next_deadline=True,
+        )
+        subtask_state = Subtask.objects.get(subtask_id=self.task_to_compute.compute_task_def['subtask_id']).state
+        self.assertEqual(subtask_state, Subtask.SubtaskState.FORCING_RESULT_TRANSFER.name)
+
+    def test_that_update_subtask_passes_validations_from_active_to_passive_state(self):
```
Comment (dybi) [inline concent_api/core/tests/test_unit_message_handlers.py]: please add assertions about `next_deadline`

File: concent_api/core/tests/test_unit_message_handlers.py (hunk 2 above)
Comment (pawelkisielewicz) [reply concent_api/core/tests/test_unit_message_handlers.py]: done

**Pull request merged**